include/zephyr/linker/common-ram.ld

```diff
@@ -12,6 +12,10 @@
 #endif
 #endif /* NETWORKING */
 
+#ifdef CONFIG_ARM_SCMI
+ITERABLE_SECTION_RAM(scmi_protocol, Z_LINK_ITERABLE_SUBALIGN)
+#endif /* CONFIG_ARM_SCMI */
+
```

Comment:
An entry in `zephyr/cmake/linker_script/common/common-rom.cmake` is required as well

Reply:
typo? did you mean `common-ram.cmake` by any chance?

Reply:
yes, RAM, sorry :)